server/index.js

```diff
@@ -0,0 +1,12 @@
+import express from 'express';
+
+const app = express(); // calling an instance of express
+const PORT = 5000; // setup PORT to be used
```

Comment:
great job, PORT might preferably be defined as an environmental variable like
PORT = 5000 || process.env.PORT

Reply:
Noted

Reply:
@allebd process.env.PORT should come first.

Reply:
Working on it

Reply:
Done